notebooks/4_SparseKernelMethods.ipynb

```diff
@@ -73,8 +74,8 @@
    "outputs": [],
    "source": [
     "## Change this cell to change the kernel function throughout\n",
-    "kernel_params = {\"kernel\": \"rbf\", \"gamma\": 1.0}\n",
-    "kernel_func = gaussian_kernel\n",
+    "kernel_params = {\"kernel\": \"precomputed\"}\n",
+    "kernel_func = rbf_kernel\n",
```

Comment:
You can use functools to define

```
from functools import partial

kernel_func = partial(rbf_kernel, gamma=1.0)
```

to avoid having to repeat gamma everywhere below

Reply:
Thank you! It works!